lib/ConnectionContextBase.ts

```diff
@@ -157,6 +162,19 @@ export module ConnectionContextBase {
       operationTimeoutInSeconds: parameters.operationTimeoutInSeconds
     };
 
+    if (parameters.config.webSocket) {
```

Comment:
I would imagine the `webSocketEndpointPath` param to drive whether websockets should be used or not.
Then the `webSocket` param to drive whether a custom websocket should be used or the default `WebSocket`.

Instead, what is happening here is that the consumers of `amqp-common` will not be able to use sockets unless they provide the `webSocket` param. And when they do provide it, we arent even using it and instead using `WebSocket`

Reply:
Definitely needed to pass the right webSocket (should be fixed now), but I think I prefer using the presence of the webSocket key to enable websockets. The websocket endpoint path could conceivably be empty for some services.

Reply:
WebSocket is used by default now.